tests/components/flo/test_binary_sensor.py

```diff
@@ -22,12 +22,17 @@ async def test_binary_sensors(hass, config_entry, aioclient_mock_fixture):
 
     assert len(hass.data[FLO_DOMAIN][config_entry.entry_id]["devices"]) == 2
 
-    valve_state = hass.states.get("binary_sensor.pending_system_alerts")
+    valve_state = hass.states.get(
+        "binary_sensor.smart_water_shutoff_pending_system_alerts"
+    )
     assert valve_state.state == STATE_ON
     assert valve_state.attributes.get("info") == 0
     assert valve_state.attributes.get("warning") == 2
     assert valve_state.attributes.get("critical") == 0
-    assert valve_state.attributes.get(ATTR_FRIENDLY_NAME) == "Pending System Alerts"
+    assert (
+        valve_state.attributes.get(ATTR_FRIENDLY_NAME)
+        == "Smart Water Shutoff Pending System Alerts"
```

Comment:
This should have become:

`"Smart water shutoff pending system alerts"`

in the new styling.

Reply:
That looks really odd to me... I would expect either (a) how it is or (b) `"Smart water shutoff pending system alerts"`. Capitalizing a word in the middle of a multi-word noun is rather unusual.

Wouldn't (b) be more appropriate?

Reply:
> That looks really odd to me...

Agree. This is the first step in untangling it all.
The frontend can later start to show it proper context instead of doing magic. The current implementation in the friendly name is mainly there to keep it sort of "backward" compatible while we sort things out.